packages/analytics-js-integrations/src/integrations/Matomo/nativeSdkLoader.js

```diff
@@ -3,7 +3,19 @@ import { LOAD_ORIGIN } from '@rudderstack/analytics-js-common/v1.1/utils/constan
 function loadNativeSdk(matomoVersion, premiseUrl, serverUrl, siteId) {
   window._paq = window._paq || [];
   (function (matomoVersion, premiseUrl, serverUrl, siteId) {
-    let u = serverUrl || '';
+    // let u = matomoVersion === 'premise' ? premiseUrl.replace('matomo.js', '') : serverUrl || '';
```

Comment:
should we remove this comment? or we can write description of the final value of u too in the comment